src/elements/featured-videos.html

```diff
@@ -113,44 +113,48 @@
     </style>
 
     <div class="container">
-      <div class="videos-header">
-        <h4 class="heading">{$ featuredVideosTitle $}</h4>
-        <div class="controls">
-          <paper-icon-button icon="icons:chevron-left" on-tap="shiftContentLeft"></paper-icon-button>
-          <paper-icon-button icon="icons:chevron-right" on-tap="shiftContentRight"></paper-icon-button>
+      <template is="dom-if" if="[[_videos]]">
```

Comment:
Well, it seems I understood.
But the thing that one component renders
- link to channel with all videos
- videos fetched from firebase

The component, I believe, should be rendered if at least one of the above is present. So I cannot just hide it if no videos exist in firebase (or path for firebase does not exist) -- there still can be link to playlist, which should be rendered and vice versa.

I can try moving firebase paths into config, but it won't help to move show/hide logic to the upper level as you suggest.